### ISSUE TITLE
store global user data as read-only

### ISSUE THREAD

**Issue by 5hanth** [amark/gun#629]
eg: `user.get(x).put(y)`

how can all nodes be able to read path `users/alias/x/y` ?

**Comment by rogowski** [comment]
Hi @5hanth !
The data saved in user context (using SEA) is signed and read-only by default.
```
user.get('x').put('y');
```

And can be read by others with: 
```
gun.user(owner_pubkey).get('x').once((v, k) => { ... } )
```

**Comment by 5hanth** [comment]
@rogowski I get that. but how to we collect aggregate values for all users. consider following entities:

Post `belongs_to` User
UserLike `belongs_to` Post
UserLike `belongs_to` User


`gun.user.get('likes').set(post_id)` will append post id to user's likes collection.

to display total likes of a post, is there a way to aggregate values submitted by all users or should like function also set a global non-user specific `likes` collection and update it's count every time user likes a post?

**Comment by rogowski** [comment]
@5hanth  Yes. Maybe the most simple way is put this data in a non-user specific data.
